cmd/csi-snapshotter/main.go

```diff
@@ -28,9 +28,11 @@ import (
 
 	"google.golang.org/grpc"
 
+	flag "github.com/spf13/pflag"
```

Comment:
Do we really need to have both `flag` and `github.com/spf13/pflag` at the same time?
`utilflag.NewMapStringBool` can work with plain `flag` too.

Reply:
yeah but `AddGoFlagSet` required pflag

Reply:
That's exactly what I was asking. Why do we need two flag sets?